### DISCUSSION TITLE
UniGetUI 3.1.2 beta 2

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#2743] [Show and tell]
![](https://img.shields.io/github/downloads/marticliment/wingetui/3.1.2-beta2/total?style=for-the-badge&color=red)

Note: This release is not digitally signed for convenience. Stable releases will always be signed.

# Changes
 - **Added cargo support**
 - Installer shouldn't crash under certain scenarios anymore
 - Improvements to icons
 - Improvements to Package Operations
 - Upgraded App Sdk 1.6
 - UI tweaks and improvements

SHA256: `27B4E938560BACC58CBD5DE4878EE7033AABA023F53226BAC4FDE0315E7DE793`

<hr /><em>This discussion was created from the release <a href='https://github.com/marticliment/UniGetUI/releases/tag/3.1.2-beta2'>UniGetUI 3.1.2 beta 2</a>.</em>

**Comment by Relax5180** [comment]
MS Defender kills AppData\Local\Temp\is-TNRC2.tmp\UniGetUI.Installer.tmp

**Comment by Dynamix69** [comment]
Even this version still has this problem:
https://github.com/marticliment/UniGetUI/issues/2698
For me on three systems (on two the actual version, on one now this beta) happening the same: no installed packages are appearing any more.

**Reply by marticliment** [reply]
running `winget list` on a command prompt returns results?

**Comment by Dynamix69** [comment]
Hi,

on all three systems appearing:

 

Fehler beim Versuch, die Quelle zu aktualisieren: winget

Fehler beim Öffnen der vordefinierten Quelle. Bitte melden Sie dieses Problem an die Entwickler von winget.

Unerwarteter Fehler beim Ausführen des Befehls:

0x80070005 : Zugriff verweigert

 

Translated:

Error when trying to update the source: winget

Error opening the predefined source. Please report this problem to the developers of winget.

Unexpected error while executing the command:

0x80070005 : Access denied

 

Not sure, why, and espially why it appeared on all three systems at the same time after wingetui worked for a while?

 

So it´s not caused by wingetui, I think?

 

Kind regards,

Mike

 

 

Von: Martí Climent ***@***.***> 
Gesendet: Montag, 16. September 2024 15:02
An: marticliment/UniGetUI ***@***.***>
Cc: Dynamix69 ***@***.***>; Comment ***@***.***>
Betreff: Re: [marticliment/UniGetUI] UniGetUI 3.1.2 beta 2 (Discussion #2743)

 

running winget list on a command prompt returns results?

**Comment by Dynamix69** [comment]
Oh, sorry, did not think twice…

I was not logged in as admin.

If I login as admin and try the command it lists all installed packages on the systems.

If I try the command winget list winget  then this comes up:

 

System 1:

Name                                       ID                                                  Version          Quelle

-----------------------------------------------------------------------------------------------------------------------

App-Installer                              Microsoft.AppInstaller                              1.23.1911.0      winget

Microsoft.UI.Xaml.2.7                      Microsoft.UI.Xaml.2.7                               7.2208.15002.0   winget

Microsoft.UI.Xaml.2.7                      Microsoft.UI.Xaml.2.7                               7.2208.15002.0   winget

Microsoft.UI.Xaml.2.8                      Microsoft.UI.Xaml.2.8                               8.2310.30001.0   winget

Microsoft.UI.Xaml.2.8                      Microsoft.UI.Xaml.2.8                               8.2310.30001.0   winget

Windows Package Manager Source (winget) V2 MSIX\Microsoft.Winget.Source_2024.812.2123.45_neut… 2024.812.2123.45

UniGetUI (formerly WingetUI)               ARP\Machine\X86\{889610CC-4337-4BDB-AC3B-4F21806C0… 3.1.1

 

 

 

System 2:

App-Installer                              Microsoft.AppInstaller                              1.23.1911.0      winget

Microsoft.UI.Xaml.2.7                      Microsoft.UI.Xaml.2.7                               7.2208.15002.0   winget

Microsoft.UI.Xaml.2.7                      Microsoft.UI.Xaml.2.7                               7.2208.15002.0   winget

Microsoft.UI.Xaml.2.8                      Microsoft.UI.Xaml.2.8                               8.2310.30001.0   winget

Microsoft.UI.Xaml.2.8                      Microsoft.UI.Xaml.2.8                               8.2310.30001.0   winget

Windows Package Manager Source (winget) V2 MSIX\Microsoft.Winget.Source_2024.912.1823.30_neut… 2024.912.1823.30

UniGetUI (formerly WingetUI)               SomePythonThings.WingetUIStore                      3.1.2-beta1      winget

 

 

 

Name                                       ID                                                  Version          Quelle

-----------------------------------------------------------------------------------------------------------------------

App-Installer                              Microsoft.AppInstaller                              1.23.1911.0      winget

Microsoft.UI.Xaml.2.7                      Microsoft.UI.Xaml.2.7                               7.2208.15002.0   winget

Microsoft.UI.Xaml.2.7                      Microsoft.UI.Xaml.2.7                               7.2208.15002.0   winget

Microsoft.UI.Xaml.2.8                      Microsoft.UI.Xaml.2.8                               8.2310.30001.0   winget

Microsoft.UI.Xaml.2.8                      Microsoft.UI.Xaml.2.8                               8.2310.30001.0   winget

Windows Package Manager Source (winget) V2 MSIX\Microsoft.Winget.Source_2024.906.1045.40_neut… 2024.906.1045.40

 

 

Kind regards,

Mike

 

 

 

Von: Martí Climent ***@***.***> 
Gesendet: Montag, 16. September 2024 15:02
An: marticliment/UniGetUI ***@***.***>
Cc: Dynamix69 ***@***.***>; Comment ***@***.***>
Betreff: Re: [marticliment/UniGetUI] UniGetUI 3.1.2 beta 2 (Discussion #2743)

 

running winget list on a command prompt returns results?

**Reply by marticliment** [reply]
Then I am afraid the issue is with WinGet, and not with UniGetUI.

To circumvent this issue, go to UniGetUI settings -> WinGet -> Use Bundled WinGet

**Comment by Biyakuga** [comment]
Thank you for the new update!
1. Is there a noticeable improvement in performance with the upgrade to App SDK 1.6?
2. Some programs are missing icons during updates. Is it possible to integrate an icon API to fill in these gaps?

**Reply by marticliment** [reply]
1. App Sdk 1.6 has some improvements both in font rendering and in window resizing.
2. You can check the icon database here: https://marticliment.com/unigetui/help/icons-and-screenshots/

**Comment by Dynamix69** [comment]
No, sorry for first message, but I corrected it later with a second one.

I was not logged in as admin into the cmd. After I was able to receive the winget details.

However I also tried to use the bundled version of winget, but it´s the same result.

 

Von: Martí Climent ***@***.***> 
Gesendet: Montag, 16. September 2024 20:02
An: marticliment/UniGetUI ***@***.***>
Cc: Dynamix69 ***@***.***>; Comment ***@***.***>
Betreff: Re: [marticliment/UniGetUI] UniGetUI 3.1.2 beta 2 (Discussion #2743)

 

Then I am afraid the issue is with WinGet, and not with UniGetUI.

To circumvent this issue, go to UniGetUI settings -> WinGet -> Use Bundled WinGet
